<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EId</class>
 <widget class="QWidget" name="EId">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_70">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_29" stretch="3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_47" stretch="0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetFixedSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="flagLabel_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../resources/librecelik.qrc">:/images/rs_flag.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_48">
             <item>
              <widget class="QLabel" name="stateLabel_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Republic Of&lt;/p&gt;&lt;p&gt;Serbia&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lkLabel_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Sans Serif</family>
               <pointsize>18</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Identity card</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Print</string>
             </property>
             <property name="icon">
              <iconset resource="../../resources/librecelik.qrc">
               <normaloff>:/images/printer-1414.png</normaloff>:/images/printer-1414.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_33">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_54">
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_34">
            <item alignment="Qt::AlignmentFlag::AlignTop">
             <widget class="QLabel" name="pictureLabel_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>240</width>
                <height>320</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../resources/librecelik.qrc">:/images/user.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Signatures of data groups</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_55">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_35" stretch="0,0">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="group1Label_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../resources/librecelik.qrc">:/images/grey_question_mark.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="documentDataLabel_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Document data</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_36">
              <item>
               <widget class="QLabel" name="group2Label_3">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../resources/librecelik.qrc">:/images/grey_question_mark.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="identityFixedDataLabel_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Personal fixed data</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_37">
              <item>
               <widget class="QLabel" name="group3Label_3">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../resources/librecelik.qrc">:/images/grey_question_mark.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="identityVarDataLabel_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Personal variable data</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_56">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="citizenGroupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Citizen Data</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_57">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_58">
              <item>
               <widget class="QLabel" name="surnameLabel_3">
                <property name="text">
                 <string>Surname</string>
                </property>
                <property name="buddy">
                 <cstring>surnameLineEdit_3</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="surnameLineEdit_3">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_59">
              <item>
               <widget class="QLabel" name="nameLabel_3">
                <property name="text">
                 <string>Name</string>
                </property>
                <property name="buddy">
                 <cstring>nameLineEdit_3</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nameLineEdit_3">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_39">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_60">
                <item>
                 <widget class="QLabel" name="nationalityLabel_3">
                  <property name="text">
                   <string>Nationality</string>
                  </property>
                  <property name="buddy">
                   <cstring>nationalityLineEdit_3</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="nationalityLineEdit_3">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_61">
                <item>
                 <widget class="QLabel" name="parentNameLabel_3">
                  <property name="text">
                   <string>Parent name</string>
                  </property>
                  <property name="buddy">
                   <cstring>parentNameLineEdit_3</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="parentNameLineEdit_3">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_40">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_62">
                <item>
                 <widget class="QLabel" name="statusOfForeignerLabel_3">
                  <property name="text">
                   <string>Status of foreigner</string>
                  </property>
                  <property name="buddy">
                   <cstring>statusOfForeignerLineEdit_3</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="statusOfForeignerLineEdit_3">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_63">
                <item>
                 <widget class="QLabel" name="placeOfBirthLabel_3">
                  <property name="text">
                   <string>Place of birth</string>
                  </property>
                  <property name="buddy">
                   <cstring>placeOfBirthLineEdit_3</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="placeOfBirthLineEdit_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_41">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_64">
                <item>
                 <widget class="QLabel" name="dateOfAddressChangeLabel_3">
                  <property name="text">
                   <string>Date of address change</string>
                  </property>
                  <property name="buddy">
                   <cstring>dateOfAddressChangeLineEdit_3</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="dateOfAddressChangeLineEdit_3">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_65">
                <item>
                 <widget class="QLabel" name="dateOfBirthLabel_3">
                  <property name="text">
                   <string>Date of birth</string>
                  </property>
                  <property name="buddy">
                   <cstring>dateOfBirthLineEdit_3</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="dateOfBirthLineEdit_3">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_66">
              <item>
               <widget class="QLabel" name="addressLabel_3">
                <property name="text">
                 <string>Address</string>
                </property>
                <property name="buddy">
                 <cstring>addressLineEdit_3</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="addressLineEdit_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_42" stretch="2,0">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_67">
                <item>
                 <widget class="QLabel" name="jmbgLabel_3">
                  <property name="text">
                   <string>JMBG</string>
                  </property>
                  <property name="buddy">
                   <cstring>jmbgLineEdit_3</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="jmbgLineEdit_3">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_68">
                <item>
                 <widget class="QLabel" name="genderLabel_3">
                  <property name="text">
                   <string>Gender</string>
                  </property>
                  <property name="buddy">
                   <cstring>genderLineEdit_3</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="genderLineEdit_3">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_31">
       <item>
        <widget class="QGroupBox" name="documentGroupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Document data</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_49">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_32">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_50">
              <item>
               <widget class="QLabel" name="documentIssuerLabel_3">
                <property name="text">
                 <string>Document issuer</string>
                </property>
                <property name="buddy">
                 <cstring>documentIssuerLineEdit_3</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="documentIssuerLineEdit_3">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_51">
              <item>
               <widget class="QLabel" name="documentNumberLabel_3">
                <property name="text">
                 <string>Document number</string>
                </property>
                <property name="buddy">
                 <cstring>documentNumberLineEdit_3</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="documentNumberLineEdit_3">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_52">
              <item>
               <widget class="QLabel" name="dateOfIssuanceLabel_3">
                <property name="text">
                 <string>Date of issuance</string>
                </property>
                <property name="buddy">
                 <cstring>dateOfIssuanceLineEdit_3</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="dateOfIssuanceLineEdit_3">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_53">
              <item>
               <widget class="QLabel" name="validToLabel_3">
                <property name="text">
                 <string>Valid to</string>
                </property>
                <property name="buddy">
                 <cstring>validToLineEdit_3</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="validToLineEdit_3">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/librecelik.qrc"/>
 </resources>
 <connections/>
</ui>
